Resources/public/js/models/ez-contentmodel.js

```diff
@@ -212,6 +212,10 @@ YUI.add('ez-contentmodel', function (Y) {
                 return this.get('relations');
             }
 
+            if (type === "ATTRIBUTE") {
```

Comment:
Also that means the first condition in the filter function below becomes useless